src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -1,7 +1,86 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
 
-const BatchGroupConfigurationSettings = () => {
-  return <div data-test-batch-group-configuration-settings />;
+import {
+  Pane,
+  Spinner,
+} from '@folio/stripes/components';
+
+import { batchGroupsResource } from '../../common/resources';
+import BatchGroupConfigurationSettingsForm from './BatchGroupCongiurationSettingsForm';
+
+const BatchGroupConfigurationSettings = ({ mutator }) => {
+  const [batchGroups, setBatchGroups] = useState();
+  const [batchGroupId, setBatchGroupId] = useState();
```

Comment:
`batchGroupId` - it's good, I know that you put id of batch group there, but for what purpose? Maybe it's a good idea to add a purpose word, something like selectedBatchGroupId or else

Reply:
Renamed it